core/src/main/java/org/polypheny/db/schema/Namespace.java

```diff
@@ -46,25 +46,25 @@
 
 /**
  * A namespace for tables and functions.
```

Comment:
Is this relational?

Reply:
https://github.com/polypheny/Polypheny-DB/pull/464#discussion_r1519277343